angular-hometask/src/app/core/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -1,15 +1,62 @@
 import { Component, OnInit } from '@angular/core';
+import { Router, NavigationEnd, ActivatedRoute, PRIMARY_OUTLET, Params, UrlSegment } from '@angular/router';
+interface Breadcrumb {
```

Comment:
IBreadCrubm

Reply:
And better create interface folder and then create file for it

Reply:
Done 951f1b8f266f42d627ea2c82f76c40ae9b8f6baa